collect_app/src/main/java/org/odk/collect/android/formentry/saving/FormSaveViewModel.java

```diff
@@ -83,7 +84,11 @@ public FormSaveViewModel(SavedStateHandle stateHandle, Clock clock, FormSaver fo
     }
 
     @Override
-    public void formLoaded(FormController formController) {
+    public void formLoaded(@NotNull FormController formController) {
+        if (formController == null) {
+            Timber.e(new IllegalArgumentException("null FormController passed to formLoaded"));
```

Comment:
My thinking here is that we can track this in non fatals in Crashlytics and hopefully gain some insight into whats happening (if this continues).